includes/wizards/class-network-wizard.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * Network (Plugin) Wizard
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Easy interface for setting up info.
+ */
+class Network_Wizard extends Wizard {
+
+	use \Newspack\Wizards\Traits\Admin_Header;
+
+	/**
+	 * Newspack Network plugin's Admin screen definitions (see below).
+	 *
+	 * @var array
+	 */
+	private $admin_screens = [];
+
+	/**
+	 * Screen type: admin page or post type.
+	 *
+	 * @var string
+	 */
+	private $screen_type = '';
+
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+
+		if ( ! is_plugin_active( 'newspack-network/newspack-network.php' ) ) {
+			return;
+		}
+		
+		// Admin screens based on Newspack Network plugin's admin pages and post types.
+		$this->admin_screens = [
+			'page'      => [
+				'newspack-network'                      => __( 'Network / Settings', 'newspack-plugin' ),
+				'newspack-network-event-log'            => __( 'Network / Event Log', 'newspack-plugin' ),
+				'newspack-network-membership-plans'     => __( 'Network / Membership Plans', 'newspack-plugin' ),
+				'newspack-network-distributor-settings' => __( 'Network / Distributor Settings', 'newspack-plugin' ),
+				'newspack-network-node'                 => __( 'Network / Node Settings', 'newspack-plugin' ),
+			],
+			'post_type' => [
+				'newspack_hub_nodes'   => __( 'Network / Nodes', 'newspack-plugin' ),
+				'np_hub_orders'        => __( 'Network / Orders', 'newspack-plugin' ),
+				'np_hub_subscriptions' => __( 'Network / Subscriptions', 'newspack-plugin' ),
+			],
+		];
+
+		// Move entire Network Menu. Use a high priority to load after Network Plugin itself loads.
+		add_action( 'admin_menu', [ $this, 'move_menu' ], 99 );
+
+		// Use current_screen for better detection of which admin screen we might be on.
+		add_action( 'current_screen', [ $this, 'current_screen' ] );
+	}
+
+	/**
+	 * Current screen callback to detect Network admin screens.
+	 *
+	 * @return void
+	 */
+	public function current_screen(): void {
+
+		global $current_screen, $plugin_page;
+
+		$set_and_show = function( $slug, $screen_type ) {
```

Comment:
@ronchambers what are your thoughts on turning this into a method i.e. `protected function set_and_show_screen( string $slug, string $screen_type ): void`, some motivations below:

- **Readability:** Improves the overall readability of the code. Instead of having to process both the logic of `current_screen()` and an anonymous function at the same time, a developer can simply refer to a clearly named method.
- **Reusability:** Useful if, in the future, we need to set up the screen in different contexts i.e. Taxonomies.
- **Testability:** We can directly unit-test `set_and_show_screen()` in isolation. anonymous functions embedded in another function would otherwise require us to write a more complex and less isolated test case.
- **Debugging and Traceability:**  If something goes wrong, a named method can be traced much more easily by developers.